container-action/scripts/configure_rit_locally.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+if [ $# -ne 1 ] || [ -z "${1:-}" ]; then
+    echo "Usage: ${0##*/} POWPEG_VERSION" >&2
+    exit 1
+fi
+POWPEG_VERSION=$1
+shift
+echo "POWPEG_VERSION received as parameter: $1"
```

Comment:
```suggestion
echo "POWPEG_VERSION received as parameter: $POWPEG_VERSION"
```

The `shift` changes the value of `$1`. In particular, given that the script aborts if it doesn't have exacly one parameter, this is accessing an unset variable. Depending on how it's run, that can be an error or a warning.